MalApi/MyAnimeListApi.cs

```diff
@@ -227,6 +305,147 @@ public Task<MalUserLookupResults> GetAnimeListForUserAsync(string user)
         }
 
         /// <summary>
+        /// Updates a user's anime list entry.
+        /// </summary>
+        /// <param name="animeId">ID of the anime</param>
+        /// <param name="updateInfo">The updated information</param>
+        /// <param name="cancellationToken"></param>
+        /// <param name="user">MAL user</param>
+        /// <param name="password">MAL password</param>
+        /// <returns></returns>
+        public async Task<string> UpdateAnimeForUserAsync(int animeId, UpdateAnimeObject updateInfo, string user, string password, CancellationToken cancellationToken)
```

Comment:
The return type should be `Task` instead of `Task<string>` with an always null string.

Reply:
Please change "user" to "username" to be more clear.